INSTALL.md

```diff
@@ -24,7 +24,96 @@ Unfortunately getting your system to this state may require multiple steps,
 the following sections describe how to install `google-cloud-cpp` on several
 platforms.
 
-### Required Libraries
+## Using `google-cloud-cpp` in CMake-based projects.
+
+Once you have installed `google-cloud-cpp` you can use the libraries from
+your own projects using `find_package()` in your `CMakeLists.txt` file:
+
+```CMake
+cmake_minimum_required(VERSION 3.5)
+
+find_package(storage_client REQUIRED)
+
+add_executable(my_program my_program.cc)
+target_link_libraries(my_program storage_client)
+```
+
+You can use a similar `CMakeLists.txt` to use the Cloud Bigtable C++ client:
+
+```CMake
+cmake_minimum_required(VERSION 3.5)
+
+find_package(bigtable_client REQUIRED)
+
+add_executable(my_program my_program.cc)
+target_link_libraries(my_program bigtable_client)
+```
+
+## Using `google-cloud-cpp` in Make-based projects.
+
+Once you have installed `google-cloud-cpp` you can use the libraries in your
+own Make-based projects using `pkg-config`:
+
+```Makefile
+GCS_CXXFLAGS   := $(shell pkg-config storage_client --cflags)
+GCS_CXXLDFLAGS := $(shell pkg-config storage_client --libs-only-L)
+GCS_LIBS := $(shell pkg-config storage_client --libs-only-l) -lcrc32c
+
+my_storage_program: my_storage_program.cc
+	$(CXXLD) $(CXXFLAGS) $(GCS_CXXFLAGS) $(GCS_CXXLDFLAGS) -o $@ $^ $(GCS_LIBS)
```

Comment:
s/CXXLD/CXX/?

[All this in the generate script I suppose.]

Reply:
Fixed.